src/infrastructure/gateway/payment/PaymentInMemoryGateway.ts

```diff
@@ -28,6 +28,8 @@ class PaymentsInMemoryGateway implements IPaymentsGateway {
     status: "paid"
   };
 
+  lastCreatePaymentRequest?: CreatePaymentRequest;
```

Comment:
this allows us to check the 'payment return url' that gets passed to the payment service.